packages/core/src/connectors/index.test.ts

```diff
@@ -88,6 +89,20 @@ describe('getConnectorInstances', () => {
     expect(connectorInstances[4]).toHaveProperty('connector', googleConnector);
     expect(connectorInstances[5]).toHaveProperty('connector', wechatConnector);
   });
+
+  test('should throw if any required connector does not exist in DB', async () => {
+    const id = 'aliyun-dm';
+    findAllConnectors.mockImplementationOnce(async () => []);
+    await expect(getConnectorInstances()).rejects.toMatchError(
+      new RequestError({ code: 'entity.not_found', id, status: 404 })
+    );
+  });
+
+  test('should access DB only once and should not throw', async () => {
+    findAllConnectors.mockClear();
```

Comment:
nit: in my opinion, each UT case should be independent. as a result, we may call mockClear after each test. As previous findAllConnectors mock implementation is specified to be only ONCE, this line may be unnecessary?

Reply:
beforeEach

Reply:
We have a coming discussion about this.
LOG-1925

Reply:
Updated, PTAL @darcyYe @simeng-li 
```ts
describe('getConnectorInstances', () => {
  beforeEach(() => {
    findAllConnectors.mockClear();
  });
……
```